padcli/command/deploy.go

```diff
@@ -63,6 +63,7 @@ func makeDeployOptions(
 		helm.NewImageProvider(
 			buildOutput.Image.String(),
 			publishOutput.PublishedImages(),
+			string(publishOutput.RegistryHost),
```

Comment:
This should not be needed. The published images have enough information to determine the host. (they are the hostname on the published image URL)

Reply:
I didn't want to do string comparison and hardcode the jetpack registry uri somewhere. I think for now this is ok.

Reply:
See comments in axiom